client/src/pages/Login.tsx

```diff
@@ -16,24 +15,26 @@ const LoginImage = styled.div`
     }
 `;
 const LoginText = styled.div`
-    color: ${colors.green};
+    color: var(--green);
     text-align: center;
     font-size: 36px;
     font-weight: 700;
```

Comment:
Same here.

Reply:
Same here we dont have `font-family` so short hand wont work.